tests/Stecman/Component/Symfony/Console/BashCompletion/CompletionHandlerTest.php

```diff
@@ -148,6 +173,10 @@ protected function createHandler($commandLine, $cursorIndex = null)
      */
     protected function getTerms($handlerOutput)
     {
+        if (null === $handlerOutput) {
```

Comment:
It should return empty array to have consistent output since it returns list of matched entries. If no completion handler was invoked, then 0 results which is equal to empty array.